it/uk/gov/hmrc/agentassurance/controllers/AgentAssuranceControllerISpec.scala

```diff
@@ -370,4 +377,87 @@ class AgentAssuranceControllerISpec extends IntegrationSpec
       response.status shouldBe 401
     }
   }
+
+  feature("/amls/create") {
+
+    val amlsCreateUrl = s"http://localhost:$port/agent-assurance/amls"
+
+    def doRequest(payload: String) =
+      Await.result(
+        wsClient.url(amlsCreateUrl)
+          .withHeaders(CONTENT_TYPE -> "application/json")
+          .post(payload), 10 seconds
+      )
+
+    def payload(maybeUtr: Option[Utr] = None,
+                supervisory: String = "supervisoryBody",
+                arn: Option[Arn] = None) = {
+      val utr = maybeUtr.getOrElse(Utr(Random.alphanumeric.take(10).mkString("")))
+      val amlsDetails = AmlsDetails(utr, supervisory, "0123456789", LocalDate.now(), arn)
```

Comment:
when Creating, the ARN should never be defined unless updated, will need to capture that in the update functionality